ursula_cli/_manifest.py

```diff
@@ -0,0 +1,11 @@
+import yaml
+
+
+class Manifest(object):
```

Comment:
Let's rename to UrsulaManifest just in case we have need for other manifests going forward.